refactor(curriculum): React basics quiz and review improvements (#58588)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/quiz-react-basics/66f1a2009e65c9a40a26d51e.md

#### --text--

> Which command is used to install a React project with Vite?

Reviewer: In my opinion, `initialize` or `bootstrap` sounds more appropriate compared to install in this context.

#### --distractors--

`npm create vite@latest my-react-app react`

---

`npx create vite@latest my-react-app -- --template react`

---

`npm install vite@latest my-react-app -- --template react`

#### --answer--

`npm create vite@latest my-react-app -- --template react`

### --question--

#### --text--

> Where are all the packages and libraries required by your project stored?

Reviewer: ```suggestion
Where does all the packages and libraries required by your project are stored?
```

Reviewer 2: The original sentence is correct, if a little awkward, but "Where does ... are stored" is not.

Perhaps
```suggestion
Where do the project's packages and libraries get installed?
```

Reviewer 3: I vote we use Oliver's suggestion. It sounds more natural

#### --text--

> Which of these is one of the problems with SPAs?

Reviewer: ```suggestion
Which of the following is a common problem with SPAs?
```

#### --text--

> With which command do you run a React project installed with Vite?

Reviewer: ```suggestion
Which of the following command starts a React project in development mode?
```

Since the command for running a React project remains the same across different build tools, I think mentioning Vite is not necessary.

#### --text--

> Which two keywords do you use to export a component?

Reviewer: ```suggestion
Which of the following sets keywords do you use in combination to export a component?
```

Reviewer: I initially wanted to suggest something like "Which of the following sets keywords do you use in combination to export an object as default?".

#### --text--

Which keyword is used to bring a component into another file?

#### --distractors--

`require`

---

`include`

---

> `include`

Reviewer 3: remove duplicate

```suggestion
`add`
```
